stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -344,7 +349,63 @@ private void getStyleFromFloat(
       Float value,
       @Nullable ViewDebug.ExportedProperty annotation,
       StyleAccumulator styles) {
-    styles.store(name, String.valueOf(value), isDefaultValue(value));
+    styles.store(name, String.valueOf(value), value == 0.0f);
+  }
+
+  private void getStyleFromBoolean(
+      String name,
+      Boolean value,
+      @Nullable ViewDebug.ExportedProperty annotation,
+      StyleAccumulator styles) {
+    styles.store(name, String.valueOf(value), false);
```

Comment:
As I look one more time, these methods themselves also do not make sense.  The if / else if blocks above would read so much more succinctly as:

    if (value instanceof Boolean) {
      styles.store(name, String.valueOf(value), false);
    } else { 
      ...